numjuggler/parser.py

```diff
@@ -19,21 +22,21 @@
     import pickle as cPickle
 
 # integer with one prefix character
-re_int = re.compile('\D{0,1}\d+')
+re_int = re.compile(r'\D{0,1}\d+')
```

Comment:
Here and in several other places: why all tepmlate strings start with ``r``? I tried both in Python2 and Python3: the re.compile() objects have the same patterns when defined with or without the ``r`` in front of the template string.

Reply:
Looks like I've already answered this: to stop Python 3.7 deprecation warnings.